pkg/api/pluginproxy/access_token_provider.go

```diff
@@ -44,9 +45,10 @@ type accessTokenProvider struct {
 }
 
 type jwtToken struct {
-	ExpiresOn       time.Time `json:"-"`
-	ExpiresOnString string    `json:"expires_on"`
-	AccessToken     string    `json:"access_token"`
+	ExpiresOn      time.Time   `json:"-"`
+	ExpiresOnValue interface{} `json:"expires_on"`
+	ExpiresIn      interface{} `json:"expires_in"`
+	AccessToken    string      `json:"access_token"`
```

Comment:
this looks like a messy way to handle an optional field.   Maybe add a function on the struct instead for getExpiresOnOrIn() or something?